[2025-04-09T08:02:02.045+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.transform_lastfm manual__2025-04-09T08:01:49.090864+00:00 [queued]>
[2025-04-09T08:02:02.055+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.transform_lastfm manual__2025-04-09T08:01:49.090864+00:00 [queued]>
[2025-04-09T08:02:02.055+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T08:02:02.118+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_lastfm> on 2025-04-09 08:01:49.090864+00:00
[2025-04-09T08:02:02.125+0000] {standard_task_runner.py:60} INFO - Started process 107 to run task
[2025-04-09T08:02:02.128+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'transform_lastfm', 'manual__2025-04-09T08:01:49.090864+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmpn1qcf5db']
[2025-04-09T08:02:02.129+0000] {standard_task_runner.py:88} INFO - Job 104: Subtask transform_lastfm
[2025-04-09T08:02:02.205+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.transform_lastfm manual__2025-04-09T08:01:49.090864+00:00 [running]> on host 55320d3f84c9
[2025-04-09T08:02:02.379+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='transform_lastfm' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T08:01:49.090864+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T08:01:49.090864+00:00'
[2025-04-09T08:02:02.380+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T08:02:02.381+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/transform_lastfm.py']
[2025-04-09T08:02:02.404+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T08:02:03.086+0000] {subprocess.py:93} INFO - ‚ùå Error al transformar Last.fm: [Errno 2] No such file or directory: '/opt/***/data/processed/lastfm.parquet'
[2025-04-09T08:02:03.203+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T08:02:03.270+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=transform_lastfm, execution_date=20250409T080149, start_date=20250409T080202, end_date=20250409T080203
[2025-04-09T08:02:03.344+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T08:02:03.467+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
